changelog/9871.bugfix.rst

```diff
@@ -0,0 +1 @@
+Handle ``ImportError`` for ``getpass`` import in ``_pytest.tempdir.get_user``
```

Comment:
Because the changelog is part of our user-facing documentation, we should describe this in terms of user-visible symptoms which are visible from the public API.  In this case, I think that means something like:

```rst
Fix a bizarre (and fortunately rare) bug where the `temp_path` fixture could raise
an internal error while attempting to get the current user's username.
```

Reply:
Thanks, done!